### PR TITLE
Add posibility to configure each step of mq separately

### PR DESCRIPTION
**Pull request by viklund**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:
Before one had to use a URL connection string to connect to the broker. With this commit it is possible to specify each of the settings separately. Also it brings the possibility to have some of the settings through environmental variables and some in the configuration file.

### CODE REVIEW

File: lega/conf/defaults.ini [hunk 1]
```diff
@@ -45,6 +45,17 @@ connection = amqps://guest:guest@mq:5672/%2F?heartbeat=0&connection_attempts=30&
...
+# ssl = true
+# username = guest
+# password = guest
+# hostname = mq
+# port = 5672
+# virtual_host = /
+# heartbeat = 0
+# connection_attempts = 30
+# retry_delay = 10
+
```
Comment (jbygdell) [inline lega/conf/defaults.ini]:
```suggestion
ssl = true
username = guest
password = guest
hostname = mq
port = 5672
virtual_host = /
heartbeat = 0
connection_attempts = 30
retry_delay = 10

```
These should probably not be commented out.

**Review by jbygdell — APPROVED** [review]
Looked at the official documentation pages and they are really out of date by now.

File: lega/conf/defaults.ini [hunk 2]
```diff
@@ -45,6 +45,17 @@ connection = amqps://guest:guest@mq:5672/%2F?heartbeat=0&connection_attempts=30&
...
+
 [db]
 connection = postgres://lega_in:secret@localhost:5432/lega
```
Comment (blankdots) [inline lega/conf/defaults.ini:60]: shouldn't we apply to DB connection parameters the same treatment ?

Comment (blankdots) [reply lega/conf/defaults.ini:60]: oh, this is for mq. My bad

**Review by blankdots — COMMENTED** [review]
LGTM, would like to see the tests pass. Do we need to redo some of the bootstrap for this ?

**Review by blankdots — APPROVED** [review]

**Pull request merged**